src/itdelatrisu/opsu/translation/I18n.java

```diff
@@ -72,26 +96,35 @@ private I18n(InputStream stream, String fallbackId) {
 
 		try (BufferedReader reader = new BufferedReader(new InputStreamReader(stream, StandardCharsets.UTF_8))) {
 			String line;
-			while ((line = reader.readLine()) != null) 
-				if (!line.isEmpty() && line.charAt(0) != 35) {
+			while ((line = reader.readLine()) != null) {
+				if (!line.isEmpty() && line.charAt(0) != '#') {
 					String[] values = line.split("=", 2);
 					this.keys.put(values[0], values[1]);
+
+					// Unicode checking
+					if (!unicode)
```

Comment:
Is there a reason why we have to do this whole unicode check at all? Couldn't we just assume that it is?